queue_services/entity-emailer/src/entity_emailer/email_processors/continuation_in_notification.py

```diff
@@ -179,10 +208,18 @@ def process(email_info: dict, token: str) -> dict:  # pylint: disable=too-many-l
 
     # assign subject
     legal_name = business.get('legalName', None)
-    if status == Filing.Status.PAID.value:
-        subject = 'Confirmation of Filing from the Business Registry'
+    if status == Filing.Status.APPROVED.value:
+        subject = 'Results of your Filing from the Business Registry'
+    elif status == Filing.Status.CHANGE_REQUESTED.value:
+        subject = 'Change Requested from the Business Registry'
     elif status == Filing.Status.COMPLETED.value:
         subject = 'Continuation Documents from the Business Registry'
+    elif status == Filing.Status.PAID.value:
+        subject = 'Confirmation of Filing from the Business Registry'
+    elif status == Filing.Status.REJECTED.value:
+        subject = 'Results of your Filing from the Business Registry'
+    elif status == 'RESUBMITTED':
+        subject = 'Confirmation of Filing from the Business Registry'
```

Comment:
```suggestion
    elif status in [Filing.Status.PAID.value, 'RESUBMITTED']:
        subject = 'Confirmation of Filing from the Business Registry'
    elif status == Filing.Status.REJECTED.value:
        subject = 'Results of your Filing from the Business Registry'
```

Reply:
Thank you. I've updated this. Please see last commit.